File: examples/java/BackgroundRemoval.java
import com.ailabtools.sdk.AILabClient;
import com.ailabtools.sdk.generated.params.CutoutUniversalBackgroundRemovalParams;
import com.ailabtools.sdk.http.FileInput;
import java.nio.file.Path;

public final class BackgroundRemoval {
    private BackgroundRemoval() {
    }

    public static void main(String[] args) throws Exception {
        String apiKey = System.getenv("AILAB_API_KEY");
        if (apiKey == null || apiKey.isEmpty()) {
            throw new IllegalStateException("Set AILAB_API_KEY before running this example.");
        }

        AILabClient client = new AILabClient(apiKey);
        var result = client.background().remove(
                CutoutUniversalBackgroundRemovalParams.builder()
                        .image(FileInput.fromPath(Path.of("image.jpg")))
                        .build());

        System.out.println(result.getData().getImageUrl());
    }
}

File: examples/dart/background_removal.dart
import 'dart:io';

import 'package:ailabtools/ailabtools.dart';

Future<void> main() async {
  final apiKey = Platform.environment['AILAB_API_KEY'];
  if (apiKey == null) {
    throw StateError('Set AILAB_API_KEY before running this example.');
  }

  final client = AILabClient(apiKey: apiKey);
  try {
    final bytes = await File('image.jpg').readAsBytes();
    final result = await client.background.remove(
      CutoutUniversalBackgroundRemovalParams(
        image: AILabFile.fromBytes(bytes, filename: 'image.jpg'),
      ),
    );
    print(result.data?.imageUrl);
  } finally {
    client.close();
  }
}
